### PR TITLE
invoice update and changes in frontend

### PR DESCRIPTION
**Pull request by ashishrawat0**
invoice api and bug removed in order api

### CODE REVIEW

File: Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
               <td class="right" width='20%'>{{ id[0] }}</td>
               <td class="right" width='20%'>{{ id[1] }}</td>
               <td class="right" width='20%'>${{ id[2] }}</td>
-              <td><i class="fas fa-eye"></i></td>
+              <td (click)="openParticularOrder(id[0])"><i class="fas fa-eye"></i></td>
```
Comment (vishivish18) [inline Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html]: why is this not running in a for loop?

File: pythonServer/backend.py [hunk 2]
```diff
@@ -331,25 +336,66 @@ def invoiceCreate():
...
+	quantityOrdered=bytes("","utf-8")
+	quantityPending=bytes("","utf-8")
+	quantityReceived=bytes("","utf-8")
+	print(invoiceData)
 	for i in range(len(invoiceDetails)):
 		itemNo=itemNo+bytes(invoiceDetails[i]['itemNo'],"utf-8")+u2py.VM
 		quantityOrdered=quantityOrdered+bytes(invoiceDetails[i]['quantityOrdered'],"utf-8")+u2py.VM
 		quantityPending=quantityPending+bytes(str(invoiceDetails[i]['quantityPending']),"utf-8")+u2py.VM
 		quantityReceived=quantityReceived+bytes(invoiceDetails[i]['quantityReceived'],"utf-8")+u2py.VM
+	invoiceData.insert(1,0,0,invoiceDate)
```
Comment (vishivish18) [inline pythonServer/backend.py:394]: Why all this magic numbers here?

File: Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html (hunk 1 above)
Comment (ashishrawat0) [reply Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html]: Id is running in for loop and we have to send the specific invoice number when  we click on it, to the update component

Comment (ashishrawat0) [reply Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html]: This have been corrected

File: Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.ts [hunk 3]
```diff
@@ -31,6 +32,7 @@ export class AllInvoicesComponent implements OnInit {
    this.invoiceService.allInvoice()
    .subscribe((res:any)=>{
      this.invoiceData=res.data
+     console.log(res)
```
Comment (ashishrawat0) [inline Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.ts]: i will remove this next time

**Comment by vishivish18** [comment]
@ashishrawat0 resolve the conflicts first

**Comment by ashishrawat0** [comment]
@vishivish18  resolved all the conflicts

**Review by lakshyakalia — APPROVED** [review]

**Pull request merged**